front-end/src/reducers/compile-contract.ts

```diff
@@ -0,0 +1,58 @@
+import EReducerState from '@/constants/reducer-state';
+
+const compileContractInitialState = {
+  isLoading: false,
+  isError: false,
+  isSuccess: false,
+  artifact: null as string | null
+};
```

Comment:
Here should we have error message beside the artifact?

Reply:
I guess not... Sometimes it can get up to 500kb / 1mb of error message...